force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -69,7 +72,10 @@ public with sharing class VariablesDataTypesOperators {
      * @return The quotient of the division, or 0 if the denominator is zero.
      */
     public static Decimal division(Integer a, Integer b) {
-        return null; // Replace null with the variable you used to store the result
+        if(b == 0){
+            return b;
+        }
+        return a / b; // Replace null with the variable you used to store the result
```

Comment:
Its a good idea to also add a null guard at the beginning of your method. You will get an error on this line if a or b is null at this point